tools/standalone_install.sh

```diff
@@ -74,8 +74,7 @@ if [ ! $(which go) ]; then
     exit 1
 fi
 go_version=$(go version)
-# This expression will need to be updated for future releases.
-echo $go_version | grep -q -E 'go version go1\.5'
+echo $go_version | grep -E -q --regexp="go[1-9]\.[5-9]"
```

Comment:
@jvehent Sorry, my bad. I didn't think thoroughly on the other test cases. If decided to keep this test, I'll see other ways to implement this.